ddtrace/llmobs/_trace_processor.py

```diff
@@ -59,18 +62,20 @@ def process_trace(self, trace: List[Span]) -> Optional[List[Span]]:
     def submit_llmobs_span(self, span: Span) -> None:
         """Generate and submit an LLMObs span event to be sent to LLMObs."""
         span_event = None
+        is_llm_span = span.get_tag(SPAN_KIND) == "llm"
+        is_ragas_integration_span = False
         try:
-            span_event = self._llmobs_span_event(span)
+            span_event, is_ragas_integration_span = self._llmobs_span_event(span)
```

Comment:
We change the signature of `_llmobs_span_event` to return whether or not the span is a evaluation integration generated span so we don't have to reparse tags / ml app from the span

Reply:
I don't love this, but okay with it for now. Is this the only place where we'll use this? If so, let's add follow-up work to extract that logic so we don't overload `_llmobs_span_event()`.